tf2/src/buffer_core.cpp

```diff
@@ -755,13 +757,25 @@ BufferCore::lookupTransform(
 void BufferCore::lookupTransformImpl(
   const std::string & target_frame,
   const std::string & source_frame,
-  const TimePoint & time, tf2::Transform & transform,
+  const TimePoint & time, tf2::Transform & transform_out,
+  TimePoint & time_out) const
+{
+  tf2::Quaternion rotation;
+  lookupTransformImpl(target_frame, source_frame, time, transform_out.getOrigin(), rotation, time_out);
```

Comment:
```suggestion
  lookupTransformImpl(target_frame, source_frame, time,
    transform_out.getOrigin(), rotation, time_out);
```

Address linter issue, "tf2.cpplint.whitespace/line_length [2] (/tmp/ws/src/geometry2/tf2/src/buffer_core.cpp:764)"